src/js/models/maps/assets/CesiumVectorData.js

```diff
@@ -15,6 +15,10 @@ define([
   AssetColorPalette,
   VectorFilters
 ) {
+  // Source: https://fontawesome.com/v6/icons/location-dot?f=classic&s=solid
+  const PIN_SVG_STRING = "<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 384 512'><!--!Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2024 Fonticons, Inc.--><path d='M215.7 499.2C267 435 384 279.4 384 192C384 86 298 0 192 0S0 86 0 192c0 87.4 117 243 168.3 307.2c12.3 15.3 35.1 15.3 47.4 0zM192 128a64 64 0 1 1 0 128 64 64 0 1 1 0-128z'/></svg>";
```

Comment:
I'm not sure if it would have an impact on rendering, but I wonder if we should/could remove the `<!--!Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2024 Fonticons, Inc.-->` part from the SVG string, and instead include it as a comment in the JS above? If we render 10,000s of pins, would it make a difference?

Reply:
Not sure how much latency it'd cause, but I did confirm the comment results in a larger encoded string on line 623. Moved to comment above